### PR TITLE
fix: use latest version of kommander, disable kubefed

### PR DESCRIPTION
**Pull request by alejandroEsc**
Should allow us to disable kubefed and its resources when using kommander chart for this chart

- Requires: https://github.com/mesosphere/charts/pull/139

### CODE REVIEW

File: stable/opsportal/values.yaml [hunk 1]
```diff
@@ -114,3 +115,6 @@ kommander:
       traefik.ingress.kubernetes.io/auth-type: forward
       traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
       traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+
+  kubefed:
```
Comment (hectorj2f) [inline stable/opsportal/values.yaml]: why does opsportal install kubefed ? shouldn't it be kommander chart who does it ?

Comment (hectorj2f) [reply stable/opsportal/values.yaml]: nvm! This is for the kommander chart.

File: stable/opsportal/values.yaml [hunk 2]
```diff
@@ -114,3 +115,13 @@ kommander:
...
       traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+
+  # disable kubefed to keep all items related to kubefed from being created
+  kubefed:
+    enabled: false
+
+###############################################################################
+#  Kubefed: Leave disabled as only kommander should deploy
+###############################################################################
+kubefed:
```
Comment (s12chung) [inline stable/opsportal/values.yaml]: I don't think this is needed because you already have it above?

Comment (s12chung) [reply stable/opsportal/values.yaml]: had a slack discussion. seems a bit weird that both definitions are needed but.... it is what it is.

**Pull request merged**